lexers/autoit.lua

```diff
@@ -2,26 +2,59 @@
 -- AutoIt LPeg lexer.
 -- Contributed by Jeff Stone.
 
-local lexer = require('lexer')
-local token, word_match = lexer.token, lexer.word_match
+local lexer = lexer
 local P, S = lpeg.P, lpeg.S
 
-local lex = lexer.new('autoit')
-
--- Whitespace.
-lex:add_rule('whitespace', token(lexer.WHITESPACE, lexer.space^1))
+local lex = lexer.new(...)
 
 -- Keywords.
-lex:add_rule('keyword', token(lexer.KEYWORD, word_match({
+lex:add_rule('keyword', lex:tag(lexer.KEYWORD, lex:word_match(lexer.KEYWORD)))
```

Comment:
Keywords were case-insensitive in the previous lexer. If they are still case-insensitive, please call `lex:word_match(lexer.KEYWORD, true)`. The same idea applies to other word lists.

Reply:
Thanks, works as expected.